shift/models/res_partner.py

```diff
@@ -88,7 +88,10 @@ class ResPartner(models.Model):
         store=True,
     )
 
-    @api.depends("cooperative_status_ids")
+    @api.depends(
+        "cooperative_status_ids",
+        "cooperative_status_ids.can_shop",
+    )
```

Comment:
```suggestion
    @api.depends("cooperative_status_ids.can_shop")
```

should be enough, no?

Reply:
agreed, fixed, will fix in other pr too